lisp/mastodon-tl.el

```diff
@@ -180,12 +180,17 @@ also render the html"
 
 (defun mastodon-tl--media (toot)
   "Retrieve a media attachment link for TOOT if one exists."
-  (let ((media-attachements (mastodon-tl--field 'media_attachments toot)))
-    (mapconcat
-     (lambda (media-attachement)
-       (let ((preview-url (cdr (assoc 'preview_url media-attachement))))
-         (mastodon-media--get-media-link-rendering preview-url)))
-     media-attachements "")))
+  (let* ((media-attachements (mastodon-tl--field 'media_attachments toot))
+         (media-string (mapconcat
+			(lambda (media-attachement)
+			  (let ((preview-url
+				 (cdr (assoc 'preview_url media-attachement))))
+			    (mastodon-media--get-media-link-rendering
+			     preview-url)))
+			media-attachements "")))
+    (if (not (equal media-string ""))
```

Comment:
Unless only accepts one branch and I want to return `""` on false rather than nil.